src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import lotto.utils.MessageUtils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public enum LottoRank {
+    FIRST(6, 2000000000),
```

Comment:
Java 7 부터 언더 스코어를 활용하여 숫자 타입의 가독성을 높일수 있게 되었습니다 ! 
```suggestion
    FIRST(6, 2_000_000_000),
```